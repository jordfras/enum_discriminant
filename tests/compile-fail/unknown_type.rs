use enum_discriminant::discriminant;

#[discriminant(unknown)]
enum EnumWithUnknownType {
    Unit = 17,
}

fn main() {}
