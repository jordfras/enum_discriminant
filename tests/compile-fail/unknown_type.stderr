error: Valid enum representation type expected as argument to the discriminant macro, e.g., #[discriminant(u8)]
 --> tests/compile-fail/unknown_type.rs:3:16
  |
3 | #[discriminant(unknown)]
  |                ^^^^^^^
